### DISCUSSION TITLE
Software

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#35601] [Other Feature Feedback, Questions, & Ideas]
Hey everyone,

Does anyone know how to host a page in GitHub?

**Comment by ho-lapang** [comment]
> Hey everyone,
> 
> Does anyone know how to host a page in GitHub?

**Comment by fadyat** [comment]
If you only have static content, **github pages** is your choice.

**Comment by iLakshya** [comment]
@fadyat 
How do we use GitHub pages?

**Reply by fadyat** [reply]
Check this https://pages.github.com/

**Comment by radar-blip** [comment]
More info here: [About GitHub Pages](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages)

**Comment by jfullstackdev** [comment]
my own tutorial, maybe you want to check this out too 

https://github.com/jdevstatic/github-pages-tutorial
